cluster/ntp.sls

```diff
@@ -3,6 +3,9 @@
 {% if grains['osmajorrelease']|int == 12 %}
 ntp:
   pkg.installed
```

Comment:
I think colons are missing here.
Let's recheck after the tests

Reply:
nope, no colons are needed there

Reply:
yes, it might be that here are needed!..